src/cl/op-node/op_node_launcher.star

```diff
@@ -162,6 +165,9 @@ def get_beacon_config(
         "--p2p.listen.ip=0.0.0.0",
         "--p2p.listen.tcp={0}".format(BEACON_DISCOVERY_PORT_NUM),
         "--p2p.listen.udp={0}".format(BEACON_DISCOVERY_PORT_NUM),
+        "--altda.enabled=" + str(da_server_context.enabled),
```

Comment:
Similar check would be needed here instead of adding it by default to all nodes irrespective of if the da server is present

Reply:
see my other comment. enabled is set to false by default, so its safe to add by default (and is what the other docker-compose devnet was doing).